src/components/keyboard/keyboard.js

```diff
@@ -1,13 +1,32 @@
 //@flow
 import BaseComponent from '../base';
+import {connect} from 'preact-redux';
+import {actions} from '../../reducers/shell';
+import {bindActions} from '../../utils/bind-actions';
+import {KeyMap} from "../../utils/key-map";
 
 /**
- * KeyboardControl component
- *
- * @class KeyboardControl
- * @extends {BaseComponent}
+ * mapping state to props
+ * @param {*} state - redux store state
+ * @returns {Object} - mapped state to this component
  */
+const mapStateToProps = state => ({
+  playerNav: state.shell.playerNav
+});
+
+const SEEK_JUMP: number = 5;
+const VOLUME_JUMP: number = 5;
+const PLAYBACK_RATES: Array<number> = [0.5, 1, 2, 4];
```

Comment:
This is duplicate of the settings `defaultSpeeds` attribute.
Better set it as store state and use it here as well.